src/DynamoCoreWpf/UI/Themes/Modern/SidebarGridStyleDictionary.xaml

```diff
@@ -0,0 +1,220 @@
+﻿<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+                    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml">
+
+    <!-- Region Control Templates-->
+
+
+    <!-- Region LibraryView + LibrarySearchView-->
+
+    <ControlTemplate x:Key="LibraryThumbTemplate"
+                     TargetType="{x:Type Thumb}">
```

Comment:
Between `LibraryThumbTemplate` and `TooltipThumbTemplate`, only `Width` value is different (one is `5` another is `7`). Hi @elayabharath, can we have the same thumb width for both library view and the tool-tip? If so we can merge these two templates into one.